### ISSUE TITLE
Make new snowflake Twitter agnostic and opensource

### ISSUE THREAD

**Issue by schrepfler** [twitter-archive/snowflake#23]

**Comment by mzakarczemny** [comment]
Any updates regarding the new version?

**Comment by lachesis** [comment]
Lol don't worry, twitter takes open source seriously! I'm sure we'll hear back from them some time this decade.

**Comment by fzyzcjy** [comment]
It has been 5 years... Any updates?

**Comment by morenoh149** [comment]
If you have to ask ...

Did you not read the readme @fzyzcjy ?

**Comment by fzyzcjy** [comment]
@morenoh149 Yes I see " working on open sourcing the next version ..." but it seems that it has been 6 years since this sentence...

**Comment by morenoh149** [comment]
So then. No updates. :)

**Comment by fzyzcjy** [comment]
Ok fine :|
